documentation/ProjectReference-Protocol.md

```diff
@@ -1,4 +1,4 @@
-﻿# The `ProjectReference` Protocol
+# The `ProjectReference` Protocol
```

Comment:
Are the changes to this file up-to-date after the changes to the protocol and interface with NuGet that were made in this PR?